aten/src/ATen/native/DispatchStub.h

```diff
@@ -106,10 +106,12 @@ struct TORCH_API DispatchStubImpl {
     std::atomic<void*> cpu_dispatch_ptr;
     void* cuda_dispatch_ptr;
     void* hip_dispatch_ptr;
+    void* habana_dispatch_ptr;
```

Comment:
No, don't do this. You will never hit DispatchStub for Habana kernels; dispatch will happen earlier in dispatcher.

Reply:
Ok, I have removed this change. Also, renamed the keys and device names to HPU.